tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -50,6 +52,13 @@ def montecarlo_radial1d(model, plasma, runner):
         runner.j_blue_estimator,
         runner.Edotlu_estimator,
     )
+
+    # Configuring the Tracking for R_Packets
```

Comment:
This is causing the Multithreading to fail. Investigation needed

Reply:
Has this been resolved?

Reply:
Yes, this has been resolved.